src/keys/utils.ts

```diff
@@ -41,18 +47,71 @@ async function generateKeyPair(bits: number): Promise<KeyPair> {
   return await generateKeyPair({ bits });
 }
 
+function generateMnemonic(words: 12 | 24 = 24): string {
+  switch (words) {
+    case 12:
+      return Bip39.generateMnemonic(128);
+    case 24:
+    default:
+      return Bip39.generateMnemonic(256);
+  }
+}
+
+function generateSeedFromMnemonic(mnemonic: string) {
```

Comment:
This is not "random". It is deterministic. We should only use `generate` prefix for things that are random to be clear.